_posts/release/2018-02-26-fluo-1.2.0.md

```diff
@@ -4,11 +4,11 @@ date: 2018-02-26 15:00:00 +0000
 version: fluo-1.2.0
 ---
 
-Apache Fluo is a distributed processing system built on Apache Accumulo.
-Fluo users can easily setup workflows that execute cross node transactions
-when data changes. These workflows enable users to continuously join new
-data into large existing data sets with low latency while avoiding
-reprocessing all data.
+With Apache Fluo, users can set up workflows that execute cross node
```

Comment:
I think the first sentence before this should be `Apache Fluo is a distributed processing system that lets users make incremental updates to large data sets`. This matches main page.